PayPal/src/test/java/com/braintreepayments/api/paypal/PayPalInternalClientUnitTest.kt

```diff
@@ -0,0 +1,1229 @@
+                any(),
+                capture(slot),
+                any(),
+                any()
+            )
+        }
+        val result = slot.captured
+        val actual = JSONObject(result)
+
+        Assert.assertEquals("INR", actual["currency_iso_code"])
+    }
+
+    @Test
+    @Throws(JSONException::class)
+    fun sendRequest_withPayPalCheckoutRequest_omitsEmptyLineItems() {
+        val braintreeClient = MockkBraintreeClientBuilder()
+            .sendPOSTSuccessfulResponse("{\"success\": true}")
+            .build()
+
+        every { merchantRepository.authorization } returns tokenizationKey
+
+        val sut = PayPalInternalClient(
+            braintreeClient,
+            dataCollector,
+            apiClient,
+            deviceInspector,
+            merchantRepository,
+            getReturnLinkUseCase,
+            setAppSwitchUseCase,
+            getAppSwitchUseCase,
+            analyticsParamRepository
+        )
+
+        val payPalRequest = PayPalCheckoutRequest("1.00", true)
+        payPalRequest.lineItems = ArrayList()
+        sut.sendRequest(context, payPalRequest, configuration, payPalInternalClientCallback)
+
+        val slot = slot<String>()
+        verify {
+            braintreeClient.sendPOST(
+                any(),
+                capture(slot),
+                any(),
+                any()
+            )
+        }
+        val result = slot.captured
+        val actual = JSONObject(result)
+        Assert.assertFalse(actual.has("line_items"))
+    }
+    @Test
+    fun sendRequest_whenRiskCorrelationIdNotNull_setsClientMetadataIdToRiskCorrelationId() {
+        every {
+            dataCollector.getClientMetadataId(
+                eq(context),
+                eq(configuration),
+                eq(true)
+            )
+        } returns "sample-client-metadata-id"
+        val braintreeClient = MockkBraintreeClientBuilder()
+            .sendPOSTSuccessfulResponse(Fixtures.PAYPAL_HERMES_RESPONSE)
+            .build()
+        every { merchantRepository.authorization } returns clientToken
+
+        val sut = PayPalInternalClient(
+            braintreeClient,
+            dataCollector,
+            apiClient,
+            deviceInspector,
+            merchantRepository,
+            getReturnLinkUseCase,
+            setAppSwitchUseCase,
+            getAppSwitchUseCase,
+            analyticsParamRepository
+        )
+
+        val payPalRequest = PayPalCheckoutRequest("1.00", true)
+        payPalRequest.riskCorrelationId = "risk-correlation-id"
+
+        sut.sendRequest(context, payPalRequest, configuration, payPalInternalClientCallback)
+
+        val slot = slot<PayPalPaymentAuthRequestParams>()
+        verify {
+            payPalInternalClientCallback.onResult(capture(slot), null)
+        }
+        val payPalPaymentAuthRequestParams = slot.captured
+        Assert.assertEquals("risk-correlation-id", payPalPaymentAuthRequestParams.clientMetadataId)
```

Comment:
```suggestion
        assertEquals("risk-correlation-id", payPalPaymentAuthRequestParams.clientMetadataId)
```

What are your thoughts on importing `Assert.assertEquals` and not having to reference `Assert` in each call?